### ISSUE TITLE
Bug: Accessibility testing fails using side nav due to contrast 

### ISSUE THREAD

**Issue by allly-b** [newjersey/njwds#24]
As a developer, using the side-nav component with nav-future set to be lighter, the default color becomes #adadad  which results in cypress a11y testing failing as the color is too light compared to the other default text colorations. 

In this context nav-future was set to be .text-base-light and the accessibility issue was changed fixed by overriding it to #72777a. Using just the uswds package didn't result in the same issue. 

Screenshot of error
![Screen Shot 2022-10-20 at 11 18 23 AM](https://user-images.githubusercontent.com/109625347/197029383-7ea4587d-0594-46eb-badd-dee9668a7b9e.png)

**Comment by timwright12** [comment]
closing due to age
